src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.constants;
+
+import edu.wpi.first.math.util.Units;
+
+public class ElevatorConstants {
+  //TODO: Set everything to meters in whole robot code
+  public static final int kMotorPort = 13;
+  public static final int kTopLimitSwitchPort = 9;
+  public static final int kBottomLimitSwitchPort = 8;
+  public static final int kAbsEncoderPort = 6;
```

Comment:
if you are not useing the absolute encoder you can remove it

Reply:
code change added